src/OpenTelemetry.Instrumentation.Wcf/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
   ([#1255](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1255))
 * Client instrumentation implementation moved to lower-level `BindingElement`.
   ([#1247](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1247))
+* Added support for non-SOAP requests.
+  ([#1251](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1251))
```

Comment:
@repl-chris Sorry I probably created a merge conflict for you by releasing the other PR on #1308 would you mind updating this file and moving the new entry up into the "Unreleased" section?

Reply:
updated 👍